transaction-client/src/main/java/org/apache/omid/tso/client/MockTSOClient.java

```diff
@@ -46,6 +51,23 @@
     }
 
     @Override
+    public TSOFuture<Long> getFence(long tableId) {
```

Comment:
Please select the right location for a comment about how you you model a fence (as a txn with begin ts = commit ts).